src/components/Review.js

```diff
@@ -0,0 +1,48 @@
+
+import React, { useState } from "react";
+import axios from 'axios';
+
+export default class Reviews extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      moviesReview: []
+    }
+  }
+  componentDidMount() {
+    const moviesId = this.props.match.params.id
+    console.log(moviesId)
+    axios.get(`
+      https://api.themoviedb.org/3/movie/${moviesId}/reviews?api_key=29d62274e14964e04e0a98c5c473c7b1&language=en-US&page=1`)
```

Comment:
same API Key is being used in multiple components. Store it in a separate file and import where required.